src/utils/mod.rs

```diff
@@ -19,7 +19,8 @@ use std::path::Path;
 use ::internal::prelude::*;
 use ::model::{EmojiIdentifier, EmojiId};
 
-pub use self::message_builder::MessageBuilder;
+pub use self::message_builder::{MessageBuilder, Content, ContentModifier};
+pub use self::message_builder::ContentModifier::*;
```

Comment:
I'm not sure this re-export is necessary, users can do this themselves if they'd like to bring all variants into scope. It's also cleaner, as if users do `use serenity::utils::*;` it'll bring the variants into scope, the other `MessageBuilder` structs+variants, `Colour`, _and_ the other functions below into scope. Instead users should do `use serenity::utils::ContentModifier::*;` if they'd like.